test/db/cmd/cmd_p_capital_c

```diff
@@ -46,9 +46,9 @@ wx 55aa55aa909055aa9090ff20000ff34534ff55f45eaac0c0c0cdf534
 pCd 3
 EOF
 EXPECT=<<EOF
-0x00000000 bge 0x156a95c                0x0000000c ldrbmi r0, [r3, 0xf00]!
-0x00000004 bge 0x156424c                0x00000010 pli [r5, -0xf34]
-0x00000008 smlalshs sb, pc, r0, r0      0x00000014 sbcgt sl, r0, lr, asr sl
+0x00000000 bge 0x156a95c                0x00000008 smlalshs sb, pc, r0, r0
+0x00000004 bge 0x156424c                0x0000000c ldrbmi r0, [r3, 0xf00]!
+0x00000008 smlalshs sb, pc, r0, r0      0x00000010 pli [r5, -0xf34]
```

Comment:
This approach looks wrong because it duplicates instructions in two different columns

Reply:
yes, I see this in test cases but when I actually run by building it is not the case i.e.:
![image](https://user-images.githubusercontent.com/75754258/228221254-8137a3c1-32e6-4ab6-8e7a-701356e92c3b.png)

should I go and change the test case or what should I follow? 

Also what is that 

> // TODO: Use the API directly
https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd/cmd_print.c#L5555

is this API related to this? if yes can you guide about that API.